tc/core/polyhedral/functional.h

```diff
@@ -119,9 +130,24 @@ std::vector<T> Filter(Func f, const std::vector<T>& input) {
   static_assert(
       function_traits<Func>::n_args == 1,
       "Filtering function must take one argument");
+
+  // Allow combinations:
+  //       T  and T
+  // const T  and T
+  // const T  and const T
+  //       T* and T*
+  // const T* and T*
+  // const T* and const T*
+  using arg0 = typename function_traits<Func>::template arg<0>::type;
+  constexpr bool sameType = std::is_same<arg0, T>::value;
+  constexpr bool sameTypeRightMbConst =
```

Comment:
What does "Mb" mean?

Reply:
"maybe"

Reply:
On Wed, Jun 20, 2018 at 02:34:14AM -0700, ftynse wrote:
> "maybe"

Hmm... it's probably not that important, but I find it difficult
to read that.  Isn't the point here that the first argument
is the const version of the second one?
How about picking a name that reflects that?
Maybe "sameTypeLeftConst".

skimo